### PR TITLE
feat: Show active file name when saving to current file

### LINKED ISSUE #3650: Save-to-current-file UI
The current solution of showing the save-icon next to darkMode when fileHandle is active is only temporary:

image

The idea is to reimplement part of @j-f1's proposal (https://github.com/excalidraw/excalidraw/pull/3381) to display active filename + save icon somewhere in the UI:

https://excalidraw.com/#json=5822182374506496,fu593dJLqIKnA22ochvbeg

### Option (A)

image

### Option (B)

image

### PR DESCRIPTION
**Pull request by h7y**
## Screenshots
Screenshot 2021-06-13 at 12 08 54 PM

### Long file name
Screenshot 2021-06-13 at 12 08 33 PM

## Closes
#3650

### CODE REVIEW

**Comment by dwelle** [comment]
Notes:

1. I'd not make it into an input at all since that suggests editability. I'm not technically against making it editable, but would leave that for later as we'd need to figure out how it should behave — rename the file (if possible)? Save to a new file handle?

2. The shape props menu should account for the fileHandle and not overflow the footer:
	
	image

**Comment by h7y** [comment]
Screenshot 2021-06-13 at 4 11 57 PM
Screenshot 2021-06-13 at 4 11 27 PM

**Comment by h7y** [comment]
1) I've changed it more to look like the UI from #3381 
2) Fixed it by changing the max height of shape props menu depending on whether active file name is shown

File: src/components/ActiveFile.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import React from "react";
+import Stack from "../components/Stack";
+import { ToolButton } from "../components/ToolButton";
+import { save, file } from "../components/icons";
+import { t } from "../i18n";
+
+type ActiveFileProps = {
+  fileName?: string;
+  onClick: () => void;
```
Comment (ad1992) [inline src/components/ActiveFile.tsx:11]: lets rename this to be `onSave` instead to be more clear

File: src/components/ActiveFile.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import "./ActiveFile.scss";
+
+import React from "react";
+import Stack from "../components/Stack";
+import { ToolButton } from "../components/ToolButton";
+import { save, file } from "../components/icons";
+import { t } from "../i18n";
```
Comment (ad1992) [inline src/components/ActiveFile.tsx:7]:
```suggestion
import React from "react";
import Stack from "../components/Stack";
import { ToolButton } from "../components/ToolButton";
import { save, file } from "../components/icons";
import { t } from "../i18n";

import "./ActiveFile.scss";
```

For consistency, as we are habituated seeing the first line `import React...` :)

**Review by ad1992 — COMMENTED** [review]
Overall LGTM 👍🏻

**Review by ad1992 — APPROVED** [review]

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
I'd tweet it alongside https://github.com/excalidraw/excalidraw/pull/3717 once merged

**Comment by ad1992** [comment]
shipping 🎉

**Pull request merged**